components/mdx.tsx

```diff
@@ -150,8 +150,18 @@ const Code = ({ children }: { children: ReactNode }) => (
   <code className="p-1">{children}</code>
 );
 
-const BaseBlock = ({ heading, color, children}: { heading: string, color: string, children: ReactNode }) => (
-  <div className={`border-l-4 border-${color}-500 shadow-sm mb-4 overflow-auto`}>
+const BaseBlock = ({
+  heading,
+  color,
+  children,
+}: {
+  heading: string;
+  color: string;
+  children: ReactNode;
+}) => (
+  <div
+    className={`border-l-4 border-${color}-500 shadow-sm mb-4 overflow-auto`}
```

Comment:
this might not work well with tailwind jit, can we list the colors we support and do:

```
className={classNames({'bg-red-500': color== 'red'})
```
?